src/pages/index.js

```diff
@@ -174,15 +170,18 @@ function handleAddCardFormSubmit(cardData) {
     });
 }
 
-function handleProfileImageForm() {
+function handleProfileImageForm(inputValues) {
   //evt.preventDefault();
+  console.log(inputValues);
   profileImageModal.setLoading(true);
   const profileImage = document.querySelector(".profile__image");
   const profileImageInput = document.querySelector("#profile-image-link");
   profileImage.src = profileImageInput.value;
```

Comment:
`profileImage.src = profileImageInput.value;`


Class `UserInfo` should set the entire user’s data (including the **avatar**)